common-lib/src/main/scala/com/gu/mediaservice/lib/config/StageIdentifier.scala

```diff
@@ -0,0 +1,17 @@
+package com.gu.mediaservice.lib.config
+
+import java.io.File
+
+import scala.io.Source.fromFile
+
+trait StageIdentifier {
```

Comment:
This trait is used in a few places to be extra cautious to only use the localstack endpoint with the AWS SDK when in DEV.